keymapper/injection/macros.py

```diff
@@ -411,6 +411,30 @@ def event(self, ev_type, code, value):
         self.tasks.append(lambda handler: handler(ev_type, code, value))
         self.tasks.append(self._keycode_pause)
 
+    def todevice(self, ev_type, code, value, device=None):
+        """Write any event to source or other device.
+
+        Parameters
+        ----------
+        ev_type: str or int
+            examples: 17, 'EV_LED', 18, 'EV_SND'
+        code : int or int
+            examples: 0, 'LED_NUML', 0, 'SND_CLICK'
+        value : int
+        device : None or str
+            device to send to (TODO)
+            None means mapping source device
```

Comment:
Same here, as long as the functionality is only boolean lets have the docstrings and parameter namings reflect exactly that. We can always extend it later. In this case the parameter is unused so lets remove it from todevice.

We can keep track of TODOs and ideas in githubs issue list instead